src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -91,12 +91,12 @@ internal abstract partial class AbstractNavigateToSearchService
             var highPriDocs = priorityDocuments.WhereAsArray(d => project.ContainsDocument(d.Id));
 
             var highPriDocsSet = highPriDocs.ToSet();
-            var lowPriDocs = project.Documents.Where(d => !highPriDocsSet.Contains(d));
+            var lowPriDocs = (await project.GetAllRegularAndSourceGeneratedDocumentsAsync(cancellationToken).ConfigureAwait(false))
+                              .Where(d => !highPriDocsSet.Contains(d));
```

Comment:
what about high-pri docs?  does that need to be updated to handle generated docs?

Reply:
also, i hate dotting off of awaited calls :)

Reply:
Yeah, for this I'm fine with the dotting off of it. :smile:

For high priority, it's not a concern _at the moment_. The high priority documents collection is the collection of active/visible documents, but we still only have source generated files opening in miscellaneous files. so we can't get a source generated document in that list. I filed a tracking bug for that work so we can put notes for that there: https://github.com/dotnet/roslyn/issues/50676

Reply:
> but we still only have source generated files opening in miscellaneous files.

This seems problematic.  How do things like Find-Refs work *inside* a SG file?

Reply:
Yeah that doesn't work; that work is scheduled for 16.10. (I was hoping for 16.9 but alas that didn't happen.)